src/datasource.test.ts

```diff
@@ -172,7 +172,7 @@ it('should provide decimation parameters correctly', async () => {
 });
 
 it('attempts to replace variables in metadata query', async () => {
-  const tableId = '${tableId}';
```

Comment:
This change doesn't actually matter but ${} is not technically the right Grafana syntax.